src/ppl/test/top_layer6.ok

```diff
@@ -7,7 +7,7 @@
 [INFO ODB-0130]     Created 54 pins.
 [INFO ODB-0131]     Created 88 components and 422 component-terminals.
 [INFO ODB-0133]     Created 54 nets and 88 connections.
-[INFO PPL-0070] Pin clk placed at (4um, 4um).
+[INFO PPL-0070] Pin clk placed at (4.8um, 4.8um).
```

Comment:
Is this intentional?   Similarly with other existing tests with diffs.

Reply:
Yes. I fixed how this was being printed to use the `dbuToMicrons` function. The previous was incorrect.